app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -52,10 +54,17 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
             marketManager
         )
 
+        binding.swipeToRefresh.apply {
+            setOnRefreshListener {
+                model.load()
+            }
+        }
```

Comment:
```suggestion
        binding.swipeToRefresh.setOnRefreshListener {
            model.load()
        }
```

no need for an apply when it's a single statement 😉

Reply:
Indeed!